src/compiler/checker.ts

```diff
@@ -7095,13 +7095,18 @@ namespace ts {
 
                 // Look up the value in the current scope
                 if (node.tagName.kind === SyntaxKind.Identifier) {
-                    valueSymbol = getResolvedSymbol(<Identifier>node.tagName);

```

Comment:
Or you can modify `getLocalSymbolForExportDefault` to not worry about export default.